tests/regression/general/issue-2172-init-empty-map.bats

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bats
+
+setup_file() {
+  load "../../bats.lib.bash"
+  load_common "env.bash"
+  load_common "gpg.bash"
+  load_common "yq.bash"
+
+  env.setup
+  gpg.setup
+
+  env.init baremetal kubespray prod
+}
+
+teardown_file() {
+  env.teardown
+  gpg.teardown
+}
+
+setup() {
+  load "../../bats.lib.bash"
+  load_assert
+  load_common "yq.bash"
+}
```

Comment:
Just for me to understand: why do you need to load `bats.lib.bash` and `yq.bash` both in `setup_file` and `setup` and why is `load_assert` not in `setup_file` instead of `setup`?

Reply:
The difference is that functions loaded in `setup_file` is only available in itself and `teardown_file` while functions loaded in `setup` is also available in test functions.
I've tried to capture this scoping [here](https://github.com/elastisys/compliantkubernetes-apps/tree/main/tests#writing-bats-tests).

So, this file be better served by only having a `setup` and `teardown` function instead of the the file variants.

Reply:
Right! That makes complete sense considering that each test is probably executed as a subprocess.